workspaces/npm/.changeset/config.json

```diff
@@ -4,6 +4,7 @@
   "commit": false,
   "fixed": [],
   "linked": [],
+  "ignore": ["app", "backend"],
```

Comment:
We should follow this instead: https://github.com/backstage/community-plugins/pull/1510#pullrequestreview-2378759390

Reply:
@karthikjeeyar Just remove this line, I will handle/try it in #1510, thanks a lot

Reply:
I'm looking into this.

But after adding

```json
  "privatePackages": {
    "tag": false,
    "version": false
  }
```

with and without updating @changeset/cli it crashes:

```
[nix-shell:~/git/backstage/community-plugins/workspaces/npm]$ yarn changeset
🦋  error TypeError: Cannot read properties of undefined (reading 'packageJson')
🦋  error     at createChangeset (/home/christoph/git/backstage/community-plugins/workspaces/npm/node_modules/@changesets/cli/dist/changesets-cli.cjs.js:416:75)
🦋  error     at add (/home/christoph/git/backstage/community-plugins/workspaces/npm/node_modules/@changesets/cli/dist/changesets-cli.cjs.js:512:26)
🦋  error     at async run (/home/christoph/git/backstage/community-plugins/workspaces/npm/node_modules/@changesets/cli/dist/changesets-cli.cjs.js:1345:5)
```

I keep you updated.

Reply:
@awanlin I will followup in #1510. Maybe we can get this in anyway and remove the ignore later when I found a solution to set `privatePackages: false` for all workspaces. :)

Reply:
removed it, thanks.